subcommittees/mentoring/minutes-2015-03-17.md

```diff
@@ -0,0 +1,208 @@
+-   Karin Lagesen
+-   Kate Hertweck
+-   Daniel Chen
+
+## Apologies
+
+-   Sheldon McKay
+-   Carol Willing
+
+## Actions/Resolutions
+
+-   RS: create calendar with meetings, debriefing sessions and others activities
+-   RS and TT: coordinate next debriefing session in the morning
+-   RH and BM: coordinate next debriefing session in the afternoon
+-   TT: write warning note for debriefing session mention the use of it for the blog post
+-   RS: create email for private feedbacks from workshops
+-   BM: write draft proposal for "Micro Workshop"
+-   TT: ask Greg about the screen record of how to setup workshop homepages.
+-   KL: write draft proposal for "Study New Instructor Experience" 
+-   RS: form for next meeting
+
+## Minutes
+
+### Administrative  Stuff
+
+KL: Who will taking notes.
+
+RS: I can do it this time.
+
+KL: Public storage of meeting notes/decisions? 
+
+ALL: Agree
+
+### Debriefing Sessions
+
+Next week session
+
+RS: Need a volunteer since Sheldon McKay will be at data carpentry hackathon.
+
+-   March 24, Tuesday, 10:00 am Eastern (UTC-5)
+    -   Raniere Silva
+    -   Tiffany can attend as well
+    -   Rayna M Harris (**BUT 12-1:15**)
+-   March 24, Tuesday, 7:00 pm Eastern (UTC-5)
+    -   Rayna M Harris
+    -   Possibly Bill (not 6-8 pm eastern, any other time
+        after noon)
+
+BM: How much time and how many participants attend it?
+
+RS: One hour and 12 participants.
+
+BM: Can we create a calendar.
+
+RS: Will do it.
+
+KL: I participate the last debriefing session. Make clear at the begin that the notes will be food for one blog post. When you say something in confidence you say it different.
+
+BM: Who will create a document?
+
+RS: Who run the workshop write the blog post.
+
+BM: Who can volunteer for the warning note?
+
+TT: I can write the warning note.
+
+TT: How we can handle private information?
+
+KL: Send email to list?
+
+RS: The list is public. I will setup another email.
+
+### "Micro" Workshop
+
+RS: Greg raised the point that many of our new instructors have not yet had the chance to teach a workshop and now a significant fraction of our instructor pool has never taught.  Greg suggested allowing these instructors to teach small workshops without oversight of an experienced instructor, but some members of steering committee had objections to this.
+
+Suggestions
+
+-   What about remote monitoring? IE, teach without direct supervision like Greg suggests, but have an experienced instructor watch a tape of the lesson to review?
+-   The biggest question here (IMO) is actually not so much us seeing how a new instructor teaching, but a new instructor seeing, from an instructor´s point of view how to prepare and teach.
+
+RS: This will be one topic on half day.
+
+KL: Teach one topic for colegues with the help of another instruction.
+
+BM: Oversight?
+
+KL: First sound a little strange. Run a small session for colegues is a good introduction/gentil other than go to a full class.
```

Comment:
“colegues” again.  And “gentit“ → “gentle”.  Although I'm not quite sure what that phrase is saying.  Maybe “… introduction.  A more gentle start than teaching a full class.”?